cypress/support/commands.ts

```diff
@@ -1,8 +1,24 @@
 /// <reference types="cypress" />
 // ***********************************************
 
+// Login via ID and cache the session cookie
+Cypress.Commands.add("login", (username, password) => {
+  const args = { username, password };
+
+  cy.session(args, () => {
```

Comment:
This was kind of nice - store the session cookie for this user/pass, so we can skip going to ID every time.

Also found I had to use a `cy.origin()` call, or it would yell at me for navigating to 2 different subdomains.